docs/theming.md

```diff
@@ -0,0 +1,297 @@
+use them to style your own components (or generic HTML elements):
+
+```css
+div.myCard {
+  background-color: var(--mdc-theme-primary);
+}
+```
+
+### Custom properties and older browsers
+
+CSS custom properties are supported on most modern browsers. However, if you need to support older browsers,
+you'll need to take a few extra steps. (These steps primarily apply to Internet Explorer 11.)
+
+1.  Load the web components polyfills, as described in the [Getting started guide](../README.md#supporting-older-browsers).
+
+2.  Load `CustomStyleInterface` (after loading the web components polyfills):
+
+    ```html
+    <script src="@webcomponents/shadycss/custom-style-interface.min.js"></script>
+    ```
+
+3.  Identify any top-level style elements that define custom properties. Add
+    a class to these style elements so you can easily locate them from script.
+
+    ```html
+    <style class="document-style">
+    html {
+      --mdc-theme-primary: brown;
+    }
+    </style>
+    ```
+
+4.  From script, call `addCustomStyle` for all document level style elements. 
+
+    ```js
+    window.ShadyCSS.CustomStyleInterface.addCustomStyle(document.querySelector('style.document-style'));
+    ```
+
+    This code assumes a single top-level style element. If you have more than one, you'll need to modify the code.
+    See [About CustomStyleInterface](https://github.com/webcomponents/polyfills/tree/master/packages/shadycss#about-customstyleinterface)
+    in the Shady CSS README for more examples of using `CustomStyleInterface`.
+
+With these steps, the polyfills provide limited support for CSS custom properties. 
+See [Limitations](https://github.com/webcomponents/polyfills/tree/master/packages/shadycss#limitations)
+in the ShadyCSS README for details.
+
+## Colors
+
+Material Design theming uses two main colors: **primary** and **secondary**. The primary color is used
+throughout most of the application and components, as the main color for your application. The secondary color is used
+for floating action buttons and other interactive elements, serving as visual contrast to the primary.
+
+In addition to the primary and secondary colors, Material Theming also defines a _background_ color for the app, 
+as well as a _surface_ color, which is used as a background in components.
+
+Finally, Material Theming has a number of text colors, which are used for rendering text and other shapes.
+
+- High-emphasis, used for the most important text.
+- Medium-emphasis, used for text which is lower in the visual hierarchy.
+- Hint, used for text hints (such as those in text fields and labels).
+- Disabled, used for text in disabled components and content.
+- Icon, used for icons.
+- On-surface, used for text that is on top of a surface background.
+- On-secondary, used for text that is on top of a secondary background.
+- On-primary, used for text that is on top of a primary background.
```

Comment:
nit: I feel like on-primary should come first since we usually list primary, then secondary
```suggestion
- On-primary, used for text that is on top of a primary background.
- On-secondary, used for text that is on top of a secondary background.
```

Reply:
Oh yeah weird. I didn't even notice I had them in that order.